.github/workflows/test.yml

```diff
@@ -21,31 +21,12 @@ jobs:
         with:
           node-version: 16
 
-      - name: Install pnpm
+      - name: PNPM Install
         id: install-pnpm
         uses: pnpm/action-setup@v2
         with:
           version: 7
-          run_install: false
-
-      - name: Get pnpm store directory
-        id: pnpm-cache
-        shell: bash
-        run: |
-          echo "STORE_PATH=$(pnpm store path)" >> $GITHUB_OUTPUT
-
-      - name: Setup pnpm cache
-        id: setup-pnpm-cache
-        uses: actions/cache@v3
-        with:
-          path: ${{ steps.pnpm-cache.outputs.STORE_PATH }}
-          key: ${{ runner.os }}-pnpm-store-${{ hashFiles('**/pnpm-lock.yaml') }}
-          restore-keys: |
-            ${{ runner.os }}-pnpm-store-
-
-      - name: Install dependencies
-        id: install-packages
-        run: pnpm install
```

Comment:
I'm not sure caching is particularly valuable in this repo - the latency between steps negates the benefits at the moment.